Add classes needed for the validator.

MichielOda opened the pull request.

MichielOda committed:
Add classes needed for the validator.

MichielOda committed:
Make UnitList internal as it could cause issues when used via dependecy instead of directly.

PedroDebevere commented on Protocol/UnitEntry.cs:
```diff
@@ -0,0 +1,34 @@
+        /// <summary>
+        /// Gets the name of the unit.
+        /// </summary>
+        public string Name { get; }
```
you can use inheritdoc now

PedroDebevere commented on Protocol/IUnitList.cs:
```diff
@@ -0,0 +1,15 @@
+﻿namespace Skyline.DataMiner.XmlSchemas.Protocol
+{
+	using System.Collections.Generic;
```
tab/space mix? add editor config file?

PedroDebevere commented on Protocol/UnitList.cs:
```diff
@@ -0,0 +1,75 @@
+        /// <summary>
+        /// Initializes a new instance of the <see cref="UnitList"/> class.
+        /// </summary>
+        /// <param name="uomFilePath">The uom file path.</param>
```
```suggestion
        /// <param name="uomFilePath">The UOM file path.</param>
```

MichielOda committed:
CR remarks

Adding editorconfig
Format document on files & csproj
Changed to inheritdoc + commit suggestion
